backend/driver.pm

```diff
@@ -23,21 +23,18 @@ use myjsonrpc;
 use signalblocker;
 use log qw(diag fctinfo);
 
+sub _collect_orphan ($session, $p, @) {
```

Comment:
Why the extra `@` args at the end?

Reply:
This function is invoked by `Mojo::RWP` and it is not clear whether this function might be called with more arguments in the future. Besides, I'm merely extending tests here; so changing parameter handling shouldn't be mixed into it (and the `@` preserves the old behavior despite using a different syntax).